### PR TITLE
Feature/add d3

### PR DESCRIPTION
**Pull request by perlo27**

### CODE REVIEW

File: src/components/Quadrant/TechItem.tsx [hunk 1]
```diff
@@ -1,26 +1,96 @@
...
+    ) {
+      return true;
+    }
+    // otherwise not
+    return false;
+  }
+
+  handleMouseOver = () =>
+    this.props.setHighlighted(this.props.technology.label);
+  handleMouseOut = () => this.props.setHighlighted(null);
```
Comment (jwfwessels) [inline src/components/Quadrant/TechItem.tsx]: new lines between these functions please.

File: src/components/Quadrant/index.tsx [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
-    display: block;
-    padding-bottom: 100%;
-  }
+//TODO: cleanup after finishing styling for radar container
 
-  @media ${MediaQueries.phablet} {
-    align-self: flex-start;
-  }
-`;
+// type CornerTypes = {
```
Comment (jwfwessels) [inline src/components/Quadrant/index.tsx]: let's delete this 😅  (I know I said we should keep but don't think we will use it again for now)

File: src/components/Quadrant/TechItem.tsx (hunk 1 above)
Comment (perlo27) [reply src/components/Quadrant/TechItem.tsx]: fixed

File: src/components/Quadrant/index.tsx (hunk 2 above)
Comment (perlo27) [reply src/components/Quadrant/index.tsx]: fixed

**Pull request merged**